07.ls_object/run_ls.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'optparse'
+
+require_relative './display'
+
+opt = OptionParser.new
+
+params = { l: false, r: false, a: false }
+opt.on('-l') { |v| params[:l] = v }
+opt.on('-r') { |v| params[:r] = v }
+opt.on('-a') { |v| params[:a] = v }
+opt.parse!(ARGV)
+input = ARGV[0] || '.'
+
+display_files = Display.new(input)
```

Comment:
```suggestion
display = Display.new(input)
```

左辺と右辺の名前は原則揃えましょう。
https://bootcamp.fjord.jp/pages/readable-variable-name

あと、inputという変数名は無味無臭なので、中身が想像できる変数名の方がいいですね。
https://twitter.com/jnchito/status/1571287095574138881

Reply:
それぞれ変数名の変更を行いました。